### PR TITLE
New: Text alignment update (fixes #362)

### LINKED ISSUE #362: Title / body / instruction alignment
### Subject of the enhancement
There isn't a clear and easy way to align the header text areas of the menu, page, article, block, or component within the framework or authoring tool without writing custom css. 

### Your environment
* 5.x FW / AAT

### Expected behaviour
The application of the header text area alignment could be optimised by having three separate drop downs containing the alignment options - in a similar vein to the vanilla's background image size, position, styles properties.

- [ ] Menu
- [ ] Page
- [ ] Article
- [ ] Block
- [ ] Component

### PR DESCRIPTION
**Pull request by guywillis**
Fixes: #362 

### New
* Added new text alignment property to page header, article, block, and component. Allows users to align title, body, or instruction elements to the left, center, or right.

### CODE REVIEW

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by eleanor-heath — APPROVED** [review]

File: example.json [hunk 1]
```diff
@@ -43,6 +48,11 @@ contentObjects.json
 
 articles.json
 "_vanilla": {
+    "_textAlignment": {
+        "_title": "left",
+        "_body": "center",
+        "_instruction": "right",
```
Comment (kirsty-hames) [inline example.json:54]: Comma needs removing. Same for other `_instruction` instances.

**Review by kirsty-hames — CHANGES_REQUESTED** [review]
Looks good to me thanks @guywillis and works as expected but I've flagged a couple of rogue commas that need removing before merging.

Also, I misunderstood the initial issue raised. I thought this was a request to align the content as one within the container. e.g. left align and vertically center text within a header etc 
I find this is something I'm constantly creating classes for. Would there be interest in this and could it work with the existing `_textAlignment` ? Meaning a new config option or extending on.

**Review by eleanor-heath — APPROVED** [review]

File: example.json (hunk 1 above)
Comment (guywillis) [reply example.json]: Done

**Review by kirsty-hames — APPROVED** [review]

**Pull request merged**